### PR TITLE
Feature company associate

### PR DESCRIPTION
**Pull request by khangaridb**

### CODE REVIEW

File: src/modules/companies/components/list/CompanyForm.js [hunk 1]
```diff
@@ -34,21 +34,23 @@ class CompanyForm extends React.Component {
...
-        this.context.closeModal();
+        document.getElementById('company-name').value = '';
+        document.getElementById('company-website').value = '';
       }
     });
   }
 
   render() {
-    const onClick = () => {
+    const onClick = e => {
```
Comment (ganzorig) [inline src/modules/companies/components/list/CompanyForm.js]: Ene render iin gadna bailgah

File: src/modules/companies/components/list/CompanyForm.js [hunk 2]
```diff
@@ -57,14 +59,14 @@ class CompanyForm extends React.Component {
         </FormGroup>
 
         <Modal.Footer>
-          <Button btnStyle="simple" onClick={onClick}>
+          <Button btnStyle="simple" onClick={e => onClick(e)}>
             <Icon icon="close" />
-            Cancel
+            Save&Close
```
Comment (ganzorig) [inline src/modules/companies/components/list/CompanyForm.js]: Save & Close geed zai awaad bich

File: src/modules/customers/components/list/CustomerForm.js [hunk 3]
```diff
@@ -35,21 +35,24 @@ class CustomerForm extends React.Component {
...
+        document.getElementById('customer-firstname').value = '';
+        document.getElementById('customer-lastname').value = '';
+        document.getElementById('customer-email').value = '';
       }
     });
   }
 
   render() {
-    const onClick = () => {
+    const onClick = e => {
```
Comment (ganzorig) [inline src/modules/customers/components/list/CustomerForm.js]: Ene bas render ees gadna bh. Gadna bhaar bolohgui bna uu?

Comment (khangaridb) [reply src/modules/customers/components/list/CustomerForm.js]: gadna bailgaj bolox baixaa anx xiixde dotr ni bichtsn bnle

**Comment by ganzorig** [comment]
CommonAccosiate.js ee ahiad joohon refactor hiimer bna. {this.state.loadmore && <LoadMore /> } ingeed shalgawal arai baga bichigleltei bolno. addTrigger dotor a tag href ntrgui zuwhun ungu uguh zorilgoor ugj bga bol uur tag dotor hiigeed css bichchih. componentWillReceiveProps dotor bga setState boolean utga {bool ? setFalse : setTrue } setState({loadMore: datas.length !== perPage}) geed bichchuul boloh sanagdaad bhin. Add hiigdsen baganii deer yun deer nemegdeed bga title maygiin yum heregtei bh. tegsh untsugt ni 5px padding tai bna deereesee 5 2 talaasaa 10px bolgochih. Search input ni ug ni fixed bwal zuger bna. renderName 2 ner niiluuldeg function olon gazar haragdaad bh shig bna. utils dotor hiigeed 1 gazraasaa duudaad ashiglahuu?

**Comment by ganzorig** [comment]
const Ids = []; iim huwisagch bna jijgeer bichih. @khangaridb

**Comment by ganzorig** [comment]
dropdown dotor a tag margin-left: 10px awsan bna. Bichsen neg css tei chin holildson bololtoi shalgaarai.

File: src/modules/companies/components/list/CompaniesList.js [hunk 4]
```diff
@@ -52,8 +52,6 @@ function CompaniesList({
           ))}
         </tbody>
       </Table>
-
-      <Pagination count={counts.all} />
```
Comment (ganzorig) [inline src/modules/companies/components/list/CompaniesList.js]: Pagination awchij bgamu?

Comment (ganzorig) [reply src/modules/companies/components/list/CompaniesList.js]: Customer bas pagination gui bolhij

File: src/modules/customers/components/list/CustomerForm.js [hunk 5]
```diff
@@ -59,18 +61,23 @@ class CustomerForm extends React.Component {
...
-            Cancel
+            Save & Close
           </Button>
 
-          <Button btnStyle="success" type="submit">
-            <Icon icon="checkmark" />
-            Save
+          <Button btnStyle="simple" onClick={() => this.context.closeModal()}>
+            <Icon icon="close" />
+            Close
```
Comment (ganzorig) [inline src/modules/customers/components/list/CustomerForm.js]: Close iig ni hamgiin ehend busad shigee tawih heregtei

**Comment by ganzorig** [comment]
@khangaridb customer company 2 pagination bhgui l bgad bna da

**Comment by ganzorig** [comment]
@khangaridb customer company 2 pagination bhgui l bgad bna daa

File: src/modules/customers/components/list/CustomersList.js [hunk 6]
```diff
@@ -30,26 +31,37 @@ const propTypes = {
...
 
     return (
       <Table whiteSpace="nowrap" hover bordered>
         <thead>
           <tr>
-            <th />
+            <th>
+              <FormControl
+                componentClass="checkbox"
+                onChange={() => this.onChange()}
```
Comment (ganzorig) [inline src/modules/customers/components/list/CustomersList.js]: Eniig constructor dotor bind hiigeed onChange={this.onChange} geed duudchwal bolmoor yum. chi  onChange={this.onChange()} ingej bicheed render hiihdee ajilaad bga bh. neg shalgaad uzeerei

**Pull request merged**